### DISCUSSION TITLE
Welcome to RecordYou Discussions!

### DISCUSSION THREAD

**Discussion by Bnyro** [Bnyro/RecordYou#205] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GuiSousa135** [comment]
Hello, I use and like Record You very much, but I would like to see the continuous recording function of audio in backgroud (like a security camera that keeps recent recordings and excludes old recordings), a very cool project for android that has this function is the Alibi recorder (https://github.com/Myzel394/Alibi). Thanksss
